nucypher/utilities/prometheus/collector.py

```diff
@@ -176,7 +178,9 @@ def initialize(self, metrics_prefix: str, registry: CollectorRegistry) -> None:
 
     def _collect_internal(self) -> None:
         application_agent = ContractAgency.get_agent(
-            PREApplicationAgent, registry=self.contract_registry
+            PREApplicationAgent,
+            self.contract_registry,
```

Comment:
Can we keep these as keyword arguments?

Reply:
Actually, `ContractAgency.get_agent()` arguments were made mandatory in #3137: https://github.com/nucypher/nucypher/pull/3137/commits/fdb28dd21e8993d70818208fe8100a53f80ccdda#diff-911a370c18d0d0e3ea62099fc5ff6daa16915da19cf5399f24e404b7d8d6d168

Reply:
Even if they're defined as mandatory, you can still uses them as kwargs. I think that's @jMyles's point.

Reply:
✅ Done